app/controllers/admin/configs_controller.rb

```diff
@@ -14,7 +14,8 @@ def create
 
       config_params.each do |key, value|
         if value.is_a?(Array)
-          SiteConfig.public_send("#{key}=", value.reject(&:blank?)) unless value.empty?
+          SiteConfig.public_send("#{key}=", value.reject(&:blank?)) unless
```

Comment:
**nitpick (non-blocking):** If we have to break it over two lines it's probably not a good use case of an inline modifier. I'd say there's enough happening in this line that we can move things into a method:

```ruby
def update_siteconfig_with_array(key, value)
  return if value.empty? && allowed_empty_enumerables.exclude?(key.to_sym)

  SiteConfig.public_send("#{key}=", value.reject(&:blank?)
end
```

```suggestion
          update_siteconfig_with_array(key, value)
```

Reply:
Great suggestion, I agree: it's getting too complicated in there!